docs/standard/microservices-architecture/architect-microservice-container-applications/distributed-data-management.md

```diff
@@ -88,19 +88,19 @@ The use of asynchronous communication is explained with additional details later
     [*https://en.wikipedia.org/wiki/Eventual\_consistency*](https://en.wikipedia.org/wiki/Eventual_consistency)
 
 -   **Data Consistency Primer**
-    [*https://msdn.microsoft.com/en-us/library/dn589800.aspx*](https://msdn.microsoft.com/en-us/library/dn589800.aspx)
+    [*https://msdn.microsoft.com/library/dn589800.aspx*](https://msdn.microsoft.com/library/dn589800.aspx)
 
 -   **Martin Fowler. CQRS (Command and Query Responsibility Segregation)**
     [*http://martinfowler.com/bliki/CQRS.html*](http://martinfowler.com/bliki/CQRS.html)
 
 -   **Materialized View**
-    [*https://msdn.microsoft.com/en-us/library/dn589782.aspx*](https://msdn.microsoft.com/en-us/library/dn589782.aspx)
+    [*https://msdn.microsoft.com/library/dn589782.aspx*](https://msdn.microsoft.com/library/dn589782.aspx)
```

Comment:
change this to https://docs.microsoft.com/azure/architecture/patterns/materialized-view